scout/load/cytoband.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from os import path
+
+from scout.utils.md5 import generate_md5_key
+from scout.utils.handle import get_file_handle
+
+LOG = logging.getLogger(__name__)
+
+
+def load_cytobands(resource, build, adapter):
+    """Parse and load cytobands from file.
+
+    Args:
+        resource(str): path to cytobands file (either build 37 or 38)
+        build(str): "37" or "38"
+        adapter(MongoAdapter)
+
+    """
+    cytobands = []
+    LOG.debug(f"Reading cytoband file for genome build {build}")
+
+    if path.exists(resource) is False:
+        LOG.error(f"Resource {resource} could not be found.")
+        return
+
+    lines = get_file_handle(resource)
+
+    for line in lines:
+        # Line will look like this:
+        # 3	58600000	63800000	p14.2	gneg
+        fields = line.split("\t")
+        chrom = fields[0]
```

Comment:
And how about ignoring "chr" here? Not an issue with current files, but does add an implicit constraint on the files.

Reply:
Done